### PR TITLE
pruned unused page components, edited Payment.tsx

### PR DESCRIPTION
**Pull request by nickBarak**

### CODE REVIEW

File: src/components/pages/Payment.tsx [hunk 1]
```diff
@@ -1,84 +1,116 @@
...
+        }
+      }
+    };
+  }
 
+  return <div id="payment_container">
+    <form id="payment_form" onSubmit={handleSubmit}>
+      <input ref={amount} type="text"  placeholder="Amount in USD" required />
+      <CardElement />
+      <button>Pay</button>
```
Comment (hishamss) [inline src/components/pages/Payment.tsx:59]: for some reason, if Stripe library didn't load, the pay button should be disabled

we need to put this back inside the pay button 
disabled={!stripe}

**Review by hishamss — CHANGES_REQUESTED** [review]
for some reason, if Stripe library didn't load, the pay button should be disabled

we need to put this back inside the pay button (payment.tsx)
disabled={!stripe}

File: src/components/pages/Payment.tsx (hunk 1 above)
Comment (nickBarak) [reply src/components/pages/Payment.tsx:59]: I don't see the point of this considering the handleSubmit function already short-circuits if (!stripe || !elements) is true

Comment (hishamss) [reply src/components/pages/Payment.tsx:59]: I agree with you

**Review by hishamss — APPROVED** [review]

**Review by ZAG852 — APPROVED** [review]

**Pull request merged**